src/resources/report.js

```diff
@@ -30,6 +32,49 @@ export default (api) =>
       }
     }
 
+    /**
+     * Save (create or update) a report.
```

Comment:
Please add a comment here saying this is overriding the default save behavior. My last comment got blown away when you squashed probably.